lib/streams.js

```diff
@@ -59,11 +59,15 @@ class BlobWriteStream extends Writable {
       if (err) return cb(err)
       this.id.blockLength = this.core.length - this.id.blockOffset
       this.id.byteLength = this.core.byteLength - this.id.byteOffset
-      this._release()
       return cb(null)
     })
   }
 
+  _destroy (cb) {
+    if (this._release) this._release()
```

Comment:
release is always set no?

Reply:
It's set on streamx `_open`, after core being ready:
https://github.com/holepunchto/hyperblobs/blob/baf4766abffa7f4cdda22f93751fc8493a585e51/lib/streams.js#L49

I think `_open` is not always called?

Reply:
👍